src/jit/assertionprop.cpp

```diff
@@ -4221,7 +4241,7 @@ void Compiler::optImpliedByCopyAssertion(AssertionDsc* copyAssertion, AssertionD
                 break;
 
             case O2K_IND_CNS_INT:
-                // This is the ngen case where we have a GT_IND of an address.
+            // This is the ngen case where we have an indirection of an address.
                 noway_assert((impAssertion->op1.kind == O1K_EXACT_TYPE) || (impAssertion->op1.kind == O1K_SUBTYPE));
```

Comment:
Merge mistake. I'll fix.

---
In reply to: [75345417](https://github.com/dotnet/coreclr/pull/6780#discussion_r75345417) [](ancestors = 75345417)